spec/features/idv/cancel_spec.rb

```diff
@@ -28,7 +28,13 @@
       hash_including(step: 'agreement'),
     )
 
-    click_on t('idv.cancel.actions.keep_going')
+    expect(page).to have_unique_form_landmark_labels
```

Comment:
I wonder if we could add this to the existing `expect_page_to_have_no_accessibility_violations` common helper? Maybe that'd start flagging issues with unrelated screens.

Not sure if it covers all the variations, but we have a spec which calls that helper for the IdV cancel screen:

https://github.com/18F/identity-idp/blob/2b7c12da4047b00077a7e083abb319d1a13a89b6/spec/features/accessibility/idv_pages_spec.rb#L16-L23